src/server/docker.rs

```diff
@@ -355,8 +355,8 @@ impl Tag {
                 Some(rev) => Some(Tag::Stable(name.into(), rev[..7].into())),
                 None => None,
             }
-        } else if name.starts_with("202") {
-            // example: `20200826052156c04ba5`
+        } else if name.starts_with("nightly_1") {
```

Comment:
```suggestion
        } else if name.starts_with("nightly_") {
```

We will not stay on `1` forever.